image_writers_guide/sti.adoc

```diff
@@ -163,6 +163,24 @@ EOF
 ----
 ====
 
+[[using-the-sti-environment-file]]
+== Using the STI environment file
+
+STI has a capability to set environment values specified in the
+`.sti/environment` file, that exists inside the application source repository.
```

Comment:
@soltysh your proposal doesn't make it clear that file goes in the application source repository.